src/main/java/org/datadog/jenkins/plugins/datadog/tracer/JavaConfigurator.java

```diff
@@ -0,0 +1,143 @@
+package org.datadog.jenkins.plugins.datadog.tracer;
+
+import hudson.FilePath;
+import hudson.model.Node;
+import hudson.util.Secret;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+import jenkins.model.Jenkins;
+import org.datadog.jenkins.plugins.datadog.clients.HttpClient;
+
+final class JavaConfigurator implements TracerConfigurator {
+
+    private static final String TRACER_DISTRIBUTION_URL = "https://dtdg.co/latest-java-tracer";
```

Comment:
I have added the possibility of overriding distribution URL via env var (configured either centrally for the Jenkins process, or via the new job property).
I would really like to push the clients to using the "latest" URL as it takes care of updating the tracer transparently for them, so I am intentionally not exposing this in UI.

Reply:
Thanks, I agree it's fine to leave it out of the UI. Do you mind adding verification around this URL to ensure it is a valid datadog/tracer URL?

Reply:
Added verification to ensure we're either downloading from the Datadog's distribution site or from Maven Central (this can be useful since Datadog does not server some minor releases). In the latter case artifact path is also checked to ensure it's the tracer that we're downloading.